shared/macros-bash.jinja

```diff
@@ -8,7 +8,7 @@ echo "Setting=$varname1" >> config_file
 #}}
 {{%- macro bash_instantiate_variables() -%}}
 {{%- for name in varargs -%}}
-populate {{{ name }}}
+{{{ name }}}="(bash-populate {{{ name }}})"
```

Comment:
I guess that you can now remove `{{# this line is intentionally left blank #}}` because you don't match the newlines anymore, but in the generated code there is quite a lot superfluous newlines.